### ISSUE TITLE
Result retains the "startvalue" of last input file when *not* using "-retain-numbering"

### ISSUE THREAD

**Issue by jaimet** [johnwhitington/cpdf-source#196]
Hi. I'm not sure whether this is a bug or whether it's intended:

I'm concatenating several "consecutive" pdfs - here's what `cpdf -print-page-labels` returns when run on each of the input files:

```
in1.pdf: <returns nothing at all>
in2.pdf: startpage=1, startvalue=4
in3.pdf: <returns nothing at all>
in4.pdf: startpage=1, startvalue=45
```

I concatenate the 4 inputs together:
`cpdf in1.pdf in2.pdf in3.pdf in4.pdf -o out.pdf`

Then inspect the result: `cpdf -print-page-labels out.pdf` returns:
```
...
startpage: 1
startvalue: 45
```
As I am not using `-retain-numbering`, I would expect the output pages to start from 1 rather than 45.

(Obviously I can work round this by using `-remove-page-labels` - it's more the surprising behaviour that I wanted to highlight).

**Comment by johnwhitington** [comment]
Can you forward me the files? It would speed up my investigation.. You can email them to john at coherentgraphics dot co uk

**Comment by johnwhitington** [comment]
I can confirm this, and have found the cause.

I haven't yet decided whether it's ok to change the behaviour, given backward compatibility concerns. I will decide for the next release.

Thankyou for bringing it to my attention.

**Comment by johnwhitington** [comment]
Reference: Pdfpagelabels.write does nothing on labels = []. We could change this to do label removal, and probably should.

**Comment by johnwhitington** [comment]
Fixed in v2.4
